### DISCUSSION TITLE
[SOLVED] Required Survey: se edita el survey, pero no deja de mostrar preguntas eliminadas

### DISCUSSION THREAD

**Discussion by ValeMP** [Cotalker/documentation#33] [4. Bug Reports] [answered]
Hola!

Estoy utilizando requires survey y resulta que cuando realizo cambios en los surveys, se siguen mostrando preguntas que fueron eliminadas.
Adjunto fotos de dos ejemplos, donde les muestro el survey como se ve en el required survey y como se ve en la seccion de edición (abierto luego de haber guardado los cambios).
![Screen Shot 2021-04-05 at 10 55 54](https://user-images.githubusercontent.com/42095617/113587975-8f65a580-95fd-11eb-8a42-c7daccce2078.png)
![Screen Shot 2021-04-05 at 10 55 43](https://user-images.githubusercontent.com/42095617/113587989-942a5980-95fd-11eb-964e-9113841dab35.png)
![Screen Shot 2021-04-05 at 10 55 24](https://user-images.githubusercontent.com/42095617/113587994-955b8680-95fd-11eb-9c19-e124d2c6d8a0.png)
![Screen Shot 2021-04-05 at 10 55 14](https://user-images.githubusercontent.com/42095617/113587997-97254a00-95fd-11eb-9e8b-ba117244e6ca.png)

**Comment by acedward** [comment]
Esto fue, al parecer, corregido en el build productivo del Jueves (01/04/2021). Pero no los releases de desarrollo.

**Comment by guillevalin** [comment, marked as answer]
@ValeMP ya está disponible en la última build de desarrollo, debes actualizar las imágenes y listo!

**Reply by ValeMP** [reply]
perfecto! se ven los cambios!
